cdk-less/impl/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as cdk from "aws-cdk-lib";
+import * as fl from "functionless";
+
+// This is just a rough sketch and called impl so we don't anchor on it.
+
+const app = new cdk.App();
+
+let stack: cdk.Stack;
+
+export const Stack = (props?: cdk.StackProps) => {
+  stack = new cdk.Stack(stack, __dirname);
+  return stack;
+};
+
+export const Function = <In, Out>(
+  closure: fl.FunctionClosure<In, Out>,
+  props?: fl.FunctionProps
+) => {
+  return new fl.Function(stack, __filename, closure);
+};
```

Comment:
It's clean, but the singleton stack is a bit of a problem.

Reply:
Really shows how easy it is to build a simpler layer on top of our foundation

Reply:
Someone didn't read the disclaimer at the top of the file :)

I have ideas on how to make this both flexible and extensible but I scoped them out of the initial sketch.